### PR TITLE
Feature: add classes

### PR DESCRIPTION
**Pull request by nachosala89**
- Set linters.
- Add Person, Student and Teacher classes.

### CODE REVIEW

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi @nachosala89  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, with an empty readme file.
Add a descriptive README file to your project - please use this [template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: teacher.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown')
+    super(age, name)
```
Comment (EricMbouwe) [inline teacher.rb]: -  [ ] You are missing here the `parent_permission` parameter in the constructor and in the `super` function as well; Indeed the Teacher Constructor extends the parent's constructor by adding **@specialization** and a parameter for it. Let's fix that

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi @nachosala89,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR ✔️
- Good commits ✔️

# Required Changes ♻️ ⛔️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: teacher.rb (hunk 1 above)
Comment (nachosala89) [reply teacher.rb]: Thank you for your suggestions! But why is it necessary to add the `parent_permission` to `Teacher` class? In the `Person` constructor is `parent_permission` added with `true` as default value. Is there any use case where the `parent_permission` will be `false` for a teacher?

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Nacho, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**